tests/NATS.Client.Core.Tests/NATS.Client.Core.Tests.csproj

```diff
@@ -22,6 +22,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <ProjectReference Include="..\..\src\NATS.Client.JetStream\NATS.Client.JetStream.csproj" />
```

Comment:
Do you think it would be better to have a separate test assembly for JetStream?  That would probably also require adding a 3rd test assembly that has all of the base setup functionality.

Reply:
Yes. Good idea. Created a `TestUtilities` project for shared functions.